fbw-common/src/wasm/systems/systems/src/navigation/adirs.rs

```diff
@@ -1962,6 +2059,38 @@ mod tests {
             ))
         }
 
+        fn baro_reference_1_hpa(&mut self, adiru_number: usize) -> Arinc429Word<f64> {
```

Comment:
Why "reference" on these ones. Better to keep the nomenclature consistent throughout I think.

Reply:
Agreed :) Fixed now.